### PR TITLE
Added tasks to construct aero_diffparm.yaml from template.

### PR DESCRIPTION
**Pull request by CoryMartin-NOAA**
# Description

# Type of change

- Bug fix (fixes something broken)
- New feature (adds functionality)
- Maintenance (code refactor, clean-up, new CI test, etc.)

# Change characteristics
- Is this a breaking change (a change in existing functionality)? YES/NO
- Does this change require a documentation update? YES/NO

# How has this been tested?

# Checklist
- [ ] Any dependent changes have been merged and published
- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [ ] I have made corresponding changes to the documentation if necessary

### CODE REVIEW

File: ush/python/pygfs/task/aero_bmatrix.py [hunk 1]
```diff
@@ -25,6 +25,7 @@ def __init__(self, config: Dict[str, Any]) -> None:
         _res_anl = int(self.config['CASE_ANL'][1:])
 
         _bmat_yaml = os.path.join(self.runtime_config.DATA, f"{self.runtime_config.CDUMP}.t{self.runtime_config['cyc']:02d}z.chem_diagb.yaml")
+        _diff_yaml = os.path.join(self.runtime_config.DATA, f"{self.runtime_config.CDUMP}.t{self.runtime_config['cyc']:02d}z.chem_diff.yaml")
```
Comment (CoryMartin-NOAA) [inline ush/python/pygfs/task/aero_bmatrix.py]: can you change all of these from diff (and DIFF) to diffusion (DIFFUSION) just to be more explicit, and not make people think it is 'difference'

File: ush/python/pygfs/task/bmatrix.py [hunk 2]
```diff
@@ -44,9 +45,31 @@ def get_bmat_config(self) -> Dict[str, Any]:
...
         """
 
-        # generate JEDI YAML file
+        # generate JEDI YAML file for stddev
         logger.info(f"Generate B Matrix YAML config: {self.task_config.bmat_yaml}")
         bmat_config = parse_j2yaml(self.task_config.BMATYAML, self.task_config, searchpath=self.gdasapp_j2tmpl_dir)
         logger.debug(f"BMAT config:\n{pformat(bmat_config)}")
 
-        return bmat_config
+        return bmat_config
```
Comment (CoryMartin-NOAA) [inline ush/python/pygfs/task/bmatrix.py]: remove the trailing whitespace here, it's causing coding norms to fail

**Comment by CoryMartin-NOAA** [comment]
@andytangborn can you fix the suggestions otherwise I can tomorrow

**Comment by andytangborn** [comment]
Sure, I can do it.

**Comment by andytangborn** [comment]
I made the changes. git push didn't do anything (??). We can discuss in the morning.

**Comment by andytangborn** [comment]
Oh sorry, one more step needed.

File: parm/config/gfs/config.aeroanlgenb [hunk 3]
```diff
@@ -9,5 +9,9 @@ echo "BEGIN: config.aeroanlgenb"
 source "${EXPDIR}/config.resources" aeroanlgenb
 
 export BMATYAML="${PARMgfs}/gdas/aero/berror/aero_diagb.yaml.j2"
+export DIFFFFYAML="${PARMgfs}/gdas/aero/berror/aero_diffusionparm.yaml.j2"
```
Comment (CoryMartin-NOAA) [inline parm/config/gfs/config.aeroanlgenb]:
```suggestion
export DIFFUSIONYAML="${PARMgfs}/gdas/aero/berror/aero_diffusionparm.yaml.j2"
```

**Pull request merged**